tests/geoip/test.yaml

```diff
@@ -0,0 +1,19 @@
+requires:
+  features:
+    - HAVE_LIBJANSSON
+    - GeoIP
```

Comment:
this looks suspicious. Should it be something like HAVE_GEOIP?

Reply:
We don't have this key. Should I add it in Suricata ?

The code is using an extension of features detection added in 0733c4e.

Reply:
Suricata in git master uses `GeoIP2`.

The changes to run.py look OK.  Its probably a good idea to extract information like this and put it in the feaures for use like this.  But GeoIP support probably does deserve a flag in the `Features:` in the first line of output.

Reply:
Indeed GeoIP2 change was needed.

Reply:
> Indeed GeoIP2 change was needed.

I'm not sure I like the use of GeoIP2 in the --build-info output.  The feature is still GeoIP to me.  The fact that its using v2 of the database is just an implementation detail that I'm not sure should bleed into the feature description.